cmd/lakefs/cmd/run.go

```diff
@@ -66,6 +66,38 @@ func checkAuthModeSupport(cfg *config.Config) error {
 	return nil
 }
 
+func GetAuthService(ctx context.Context, cfg *config.Config, logger logging.Logger, kvStore kv.Store) auth.Service {
```

Comment:
Maybe `CreateAuthService`, to be explicit that this is generated (and not fetched)?